core_data_modules/traced_data/io.py

```diff
@@ -439,6 +443,214 @@ def import_coda_to_traced_data_iterable_as_matrix(cls, user, data, key_of_raw, c
             td.append_data(td_matrix_data, Metadata(user, Metadata.get_call_location(), time.time()))
 
 
+class TracedDataCoda2IO(object):
+    @classmethod
+    def compute_message_ids(cls, user, data, raw_message_key, message_id_key_to_write):
```

Comment:
I think s/raw_message_key/message_key/ would make this more consistent with the other aspects.